devfile/ready-to-use.devfile.yaml

```diff
@@ -0,0 +1,196 @@
+specVersion: 0.0.1
+name: chectl-generated
+projects:
+  - source:
+      type: git
+      location: 'https://github.com/sleshchenko/NodeJS-Sample-App.git'
+    name: NodeJS-Sample-App.git
+components:
+  - type: kubernetes
+    alias: node-js
```

Comment:
I would name it differently, more focus on what it is: the user application. What about `employee-manager-app` ?

Reply:
I like it. Will change!